packages/plugin-infra/src/web3/useFungibleToken.ts

```diff
@@ -11,7 +11,7 @@ export function useFungibleToken<S extends 'all' | void = void, T extends Networ
     const connection = useWeb3Connection(pluginID, options)
 
     return useAsyncRetry<Web3Helper.FungibleTokenScope<S, T> | undefined>(async () => {
-        if (!connection) return
-        return connection?.getFungibleToken?.(address ?? '', options)
+        if (!connection || !address) return
+        return connection?.getFungibleToken?.(address, options)
```

Comment:
Not all chains rely on address, e.g Solana NFT.

Please revert this and leave a comment :

```ts
// Solana NFT doesn't require address.
```